mne/epochs.py

```diff
@@ -1556,13 +1556,10 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             If 'truncate', events will be truncated from the end of each event
             list. If 'mintime', timing differences between each event list
             will be minimized.
-        copy : bool
-            If True, a copy of epochs will be returned. Otherwise, the
-            function will operate in-place.
```

Comment:
yes, don't touch `copy` for now. The logic behind returning `self` is that you can do chaining operations. So, something like `epochs.equalize_epoch_counts().average()` should work easily. Also, @kaichogami you cannot remove an argument from a public function without deprecating it first. So, let's not touch `copy`. Just return `self` whether or not `copy` is `True` or `False` and you have my +1.

@jona-sassenhagen where is it inconsistent? Separate issue maybe?